cgyle/proxy.py

```diff
@@ -161,30 +161,37 @@ def create_local_distribution_instance(
                 username, password = proxy_creds.split(':')
             except ValueError:
                 raise CgyleCredentialsError(
-                    f'Invalid credentials, expected user:pass, got {proxy_creds}'
+                    'Invalid credentials, expected user:pass, got {}'.format(
```

Comment:
ah yeah because I disliked the line break caused by the long string when expressed as f-string. format string with inline expression compared to format string with format() expression is actually the same thing. For consistency reasons you are right of course, but this is just me and my editor who dislikes the annoying line breaks after 80 characters, and yes this is old school and I accept pull requests from other people producing longer lines, but for myself I still stick to this 1980 behavior ... yes I'm an old butt :smile: